ha/fault_tolerance/fault_tolerance_driver.py

```diff
@@ -14,42 +14,57 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
+"""
+   Set of periodically executable routines which accepts fault events from
+   message bus and takes further action on it
+"""
+
 
 import time
 
 from cortx.utils.conf_store import Conf
 from cortx.utils.log import Log
+from cortx.utils.message_bus import MessageConsumer
```

Comment:
we should use the message_bus wrpper avilable in HA instead of using the message bus from utils directly

Reply:
Done